src/wanderer.ts

```diff
@@ -276,6 +323,19 @@ const wandererTargets = [
       return have(guzzlrBooze);
     }
   ),
+  new WandererTarget(
+    "Yellow Ray",
+    (type: DraggableFight) => type === "yr",
+    () => {
+      const bestLocation = bestYrLocation();
+      if (bestLocation.value > 0) {
+        return bestLocation.location;
+      } else {
+        return null;
+      }
```

Comment:
I feel like this should be ternary iff a linter doesn't split it up over multiple lines.